jest.config.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
   reporters: ["default", "jest-junit"],
-  projects: ["<rootDir>/packages/augur-node", "<rootDir>/packages/augur-state", "<rootDir>/packages/augur-ui"]
+  projects: [
+    "<rootDir>/packages/augur-state",
+    // "<rootDir>/packages/augur-ui",
```

Comment:
this works for me

Reply:
FWIW, these tests will be back online after I get the UI building properly in the monorepo.